force-app/main/default/classes/DBStatisticCalloutService.cls

```diff
@@ -0,0 +1,38 @@
+public with sharing class DBStatisticCalloutService {
+    public static HttpResponse getSObjectAPINames() {
+        Http http = new Http();
+        HttpRequest request = new HttpRequest();
+        String authValue = 'Bearer '+ UserInfo.getSessionId();
+        request.setEndpoint('https://techmagic22-dev-ed.develop.my.salesforce.com/services/apexrest/api/SObjects');
```

Comment:
please get Org Domain Url dynamically, replace in all places where hard-coded URL is used